src/v2/components/ChannelTableContents/components/ChannelRow/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { forwardRef } from 'react'
```

Comment:
Every table row needs to be a forwardRef component so that we can attach an IntersectionObserver to it